addons/account/views/account_tax_views.xml

```diff
@@ -19,6 +19,18 @@
             </field>
         </record>
 
+        <record id="view_onboarding_tax_tree" model="ir.ui.view">
+            <field name="name">account.onboarding.tax.tree</field>
+            <field name="model">account.tax</field>
+            <field name="inherit_id" ref="account.view_tax_tree" />
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//tree" position="attributes">
+                    <attribute name="default_order">active desc, type_tax_use desc, amount desc, sequence</attribute>
```

Comment:
Why is it not done in the default `_order` of the python model directly?

Reply:
Because this order should be presented to the user only during onboarding when he's reviewing the taxes.
![image](https://user-images.githubusercontent.com/91618364/169266660-2401f553-129a-468e-b4c3-5a01271a2da1.png)

So the default order should be kept if he goes in Config > Taxes.